libbeat/docs/release.asciidoc

```diff
@@ -0,0 +1,15 @@
+:beats-dir: ../../../beats
+
+== Release notes
+
+--
+This section summarizes the changes in each release.
+
+* <<release-notes-1.0.0>>
+* <<release-notes-1.0.0-rc2>>
+* <<release-notes-1.0.0-rc1>>
+* <<release-notes-1.0.0-beta4>>
+
+--
+
+include::{beats-dir}/CHANGELOG.asciidoc[]
```

Comment:
So this only includes the most recent release? Wouldn't we have then to specific the "tag"? http://asciidoctor.org/docs/user-manual/#selecting-parts-of-a-document-to-include (I didn't test if it works).